corehq/apps/app_manager/suite_xml.py

```diff
@@ -1099,7 +1099,7 @@ def _case_tile_template_string(self):
         with open(os.path.join(
                 os.path.dirname(__file__), "case_tile_templates", "tdh.txt"
         )) as f:
-            return f.read()
+            return unicode(f.read())
```

Comment:
This was being used here: https://github.com/dimagi/commcare-hq/pull/5657/files#diff-27910c0ed3e4d6d075211ffc4db6cab4R1090

This fixed it as it was returning a str object, and now returns a unicode object. The format was using unicode parameters which caused format to fail. 

It looks like unicode(f.read()) and f.read().decode("utf-8") return the same unicode object. Is decode("utf-8") a better solution to that issue? If so, I'll change it over.